spec/takeaway_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:burger_description) {"Cheeseburger"}
+  let(:burger_price) {4.99}
+  let(:burger_to_s) {"#{burger_description}\t#{burger_price}"}
+  let(:burger) { double :dish, :description => burger_description,
+                              :price => burger_price,
+                              :to_s => burger_to_s }
+  let(:kebab_description) {"Kebab"}
+  let(:kebab_price) {3.00}
+  let(:kebab_to_s) {"#{kebab_description}\t#{kebab_price}"}
+  let(:kebab) { double :dish, :description => kebab_description,
+                              :price => kebab_price,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)